tsconfig.json

```diff
@@ -1,22 +1,21 @@
 {
   "compilerOptions": {
-    "target": "ES2020",
```

Comment:
Again, not sure if these changes belong in this PR? Probably left over from out pkgu experiment, which is now in a separate branch.

Reply:
Bump! 😄

Reply:
Reverted to keep tsconfig.json same as on main